src/types.ts

```diff
@@ -20,6 +20,7 @@ export interface PlayerType {
   nametag: string | null;
   displayName: string;
   connectCode: string;
+  slippiUid: string;
```

Comment:
Can we just call this `userId`?
```suggestion
  userId: string;
```